### ISSUE TITLE
Mod Incompatibility: Controllable.

### ISSUE THREAD

**Issue by IC-Dyllac** [CreativeMD/CreativeCore#143]
I'm sorry, I know this isn't the right page, but I saw no report issue button on the LittleTiles page.

I want to inform of an incompatibility between LittleTiles v1.5.57 and Controllable 0.11.2 [Link](https://www.curseforge.com/minecraft/mc-mods/controllable) 

Compatibility: Incompatible. Game crashes at launch when both mods are loaded.

Tested with:
- Creative Core v1.10.70
- Forge v14.23.5.2859
- Minecraft 1.12.2

If you need any more info let me know, if not feel free to close the issue when you read it, as I only wanted to inform of the incompatibility since I didn't see it mentioned on the [LittleTiles compatibility wiki.](https://github.com/CreativeMD/LittleTiles/wiki/Mod-Compatibility)

**Comment by CreativeMD** [comment]
Too Bad. The wiki list is outdated. We really need to rework the wiki, but no idea when we will get to it (most likely after LittleTiles has been ported to 1.18). I have also dropped support for 1.12, so I can't fix it. Nonetheless a crash-report is necessary to figure out the cause of the issue, so please post it as well if you report something. Not sure if Controllable still has support for 1.12, maybe it's worth a try to ask over there.

**Comment by IC-Dyllac** [comment]
It's ok. I know 1.12.2 it's not maintained on LittleTiles nor Controllable (which is why I decided not posting a crash report) and honestly I don't need a fix.  I just reported so that the incompatibility is know and for those that play on 1.12.2 can easily know it. (Since I assume not many people try to interpret crash logs.)
I'll post a log for 1.12.2 version anyways. There is no crash-log since doesn't even "start" 
Also if porting to 1.18 it might be good to keep in mind of the current incompatibility since Controllable is already on 1.18.
[latest.log](https://github.com/CreativeMD/CreativeCore/files/8597098/latest.log)

Thank you for taking the time to read this.
